src/main/js/apps/sample/app.json

```diff
@@ -42,6 +42,7 @@
     "bundles": {
         "dn_elevationprofile": {
             "Config": {
+                "startSketchOnOpen": true,
```

Comment:
Vllt in der Sample-App deaktivieren, da es der default ist?

Reply:
Eledigt